draft-nuttycom-funding-allocation.rst

```diff
@@ -171,12 +171,168 @@ complete set of funding streams will be::
+     community members will be driven to propose and execute projects that
+     enhance the network, knowing that successful initiatives have the
+     potential to receive funding. This model fosters a culture of ongoing
+     innovation and development, ensuring that Zcash remains at the forefront
+     of blockchain technology.
+
+   - **Aligning Long-Term Interests**: By tying a significant portion of the
+     block reward to future decentralized grants funding, the model aligns
+     the long-term interests of all stakeholders. Miners, developers, and
+     community members alike have a vested interest in maintaining and
+     improving the Zcash network, as the value and success of their efforts
+     are directly linked to the availability and effective use of the lockbox
+     funds. This alignment of incentives ensures that the collective efforts
+     of the community are focused on the sustainable growth and advancement
+     of the Zcash ecosystem.
+
+Guidance on Future Requirements for Alternative 1
+'''''''''''''''''''''''''''''''''''''''''''''''''
+
+To support the motivations outlined, the following guidance is proposed for
+Alternative 1. Future ZIP(s) will define the disbursement mechanism. These
+are suggestions to achieve the outlined motivations that should be considered
+in those future ZIP(s):
+
+1. **Cap on Grants**: Grants should be capped to prevent any single project
+   from depleting the lockbox funds rapidly. This ensures long-term
+   availability of funds for various projects. Additionally, capping grants
+   necessitates more granular definitions of work, goals, milestones, KPIs, and
+   achievements, requiring that work be confirmed and approved in smaller, more
+   manageable increments.
+
+2. **Decentralized Disbursement Mechanism**: The disbursement mechanism should
+   be sufficiently decentralized to ensure the regulatory motivations are
+   fulfilled. A decentralized mechanism could reduce the need for recipients to
+   undergo KYC with a controlling party, preserving privacy and aligning with
+   the ethos of the Zcash project.
+
+3. **Governance and Accountability**: The governance structure for the
+   disbursement mechanism should be open and accountable, with decisions made
+   through community consensus or decentralized voting processes to maintain
+   trust and accountability. This approach will help ensure that the allocation
+   of funds is fair and aligned with the community's evolving priorities.
+
+4. **Periodic Review and Adjustment**: There should be provisions for periodic
+   review and adjustment of the funding mechanism to address any emerging
+   issues or inefficiencies and to adapt to the evolving needs of the Zcash
+   ecosystem. This could include the ability to add or remove participants as
+   necessary. Regular assessments will help keep the funding model responsive
+   and effective, ensuring it continues to meet the community's goals.
+
```

Comment:
Possible period where funds have to sit for a specified amount of time like 3,4,6,12 months? IDK if it's worth mentioning. I guess there are a lot of potential options and mechanisms.... where is the line between ideas and guidance? ...